build_projects/shared-build-targets-utils/Utils/DotNetCli.cs

```diff
@@ -4,20 +4,22 @@
 using System.Runtime.InteropServices;
 using Microsoft.DotNet.Cli.Build.Framework;
 using Microsoft.DotNet.InternalAbstractions;
+using System.Collections.Generic;
 
 namespace Microsoft.DotNet.Cli.Build
 {
-    public class DotNetCli
+    public partial class DotNetCli
     {
         public static readonly DotNetCli Stage0 = new DotNetCli(GetStage0Path());
-        public static readonly DotNetCli Stage1 = new DotNetCli(Dirs.Stage1);
-        public static readonly DotNetCli Stage2 = new DotNetCli(Dirs.Stage2);
 
         public string BinPath { get; }
+        public IEnumerable<string> SharedFxVersionDirectories { get; private set; }
```

Comment:
fixed.

---
In reply to: [66351900](https://github.com/dotnet/core-setup/pull/97#discussion_r66351900) [](ancestors = 66351900,66350264)